migrations/036-add_customers_table.js

```diff
@@ -0,0 +1,32 @@
+var db = require('./db')
+
+exports.up = function(next) {
+  const sql =
+    [
+    'CREATE TABLE customers ( ' +
```

Comment:
Use ES2015 template literals here and in compliance_authorizations: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

Example: https://github.com/lamassu/lamassu-server/blob/v5/migrations/035-log_bank_notes.js